napari/layers/points/_tests/test_points.py

```diff
@@ -2502,3 +2488,34 @@ def test_set_drag_start():
     np.testing.assert_array_equal(layer._drag_start, position)
     layer._set_drag_start({0}, position=(1, 2))
     np.testing.assert_array_equal(layer._drag_start, position)
+
+
+def test_point_slice_request_response():
```

Comment:
As we're reimplementing `set_view_slice` with request + response, we should get test coverage through that, but adding explicit tests is a good addition.